packages/odyssey-react-mui/src/Callout.tsx

```diff
@@ -29,7 +31,17 @@ export type CalloutProps = {
   /**
    * The contents of the Callout
    */
-  children: ReactNode;
+  children?: ReactNode;
+  /**
+   * If linkUrl is not undefined, this is the text of the link.
+   * If left blank, it defaults to "Learn more".
+   * Note that linkText does nothing if linkUrl is not defined
+   */
+  linkText?: string;
+  /**
+   * If defined, the Toast will include a link to the URL
+   */
+  linkUrl?: string;
```

Comment:
If I understand correctly, if `linkUrl` exists, so should `linkText` should maybe be set?
Here's a possible suggestion for you to get TS to respect this:
```
export type CalloutProps = {
  // existing props...
} & (
  | {
      linkUrl: string;
      linkText?: string;
    }
  | {
      linkUrl?: never;
      linkText?: never
    }
)
```
This way, if `linkUrl` is present, the component may accept a `linkText` as well, optionally. Otherwise, TS will throw an error if say, someone provides `linkText` but no `linkUrl`